### ISSUE TITLE
Bad accuracy in custom dataset.

### ISSUE THREAD

**Issue by phambao** [4uiiurz1/keras-arcface#6]
I have run your code and when I changed datasets, the accuracy is low about 0.01 with both CosFace and ArcFace loss. But when I change softmax loss, accuracy raises 0 -> 0.8 after 10 epochs. How could that happen?
